chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserSubscriptionDispatcher.kt

```diff
@@ -48,4 +53,24 @@ internal class UserSubscriptionDispatcher(
                 dispatcher(RoomDeleted(event.roomId))
         }
     }
+
+    override fun onSubscribe() {
+        // Not yet implemented
+    }
+
+    override fun onOpen(headers: Headers) {
+        // Not yet implemented
+    }
+
+    override fun onRetrying() {
+        // Not yet implemented
+    }
+
+    override fun onError(error: Error) {
+        // Not yet implemented
+    }
+
+    override fun onEnd(error: EOSEvent?) {
```

Comment:
will we need EOSEvent? maybe following the feedback theme above likewise it should be just simply Error?

Reply:
I could unwrap it to an `EosError` and drop the status code and headers - `EosError` is a subtype of `Error`

Reply:
yeah that would be a nice little simplification, but aren't there any chances any of that comes useful in the future?

Reply:
there's Retry-After header though probably it does not apply to when sub is ended?